### PR TITLE
Feat: admin user first login need update password && email

### LINKED ISSUE #414: Forced to modify the admin user password after first login.
In the 1.3 release, the VelaUX creates a default admin user, after login success, the user is required to change the password and add the email address.

### PR DESCRIPTION
**Pull request by wangbow**
Signed-off-by: wb-wb665667 <wb-wb665667@alibaba-inc.com>

### Description of your changes

Fixes #414 

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary.
- [x] Run `yarn lint` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+        email,
+        password,
+      };
+      this.setState({
+        isLoading: true,
+      });
+      updateUser(params)
+        .then((res) => {
+          if (res) {
+            Message.success(<Translation>Update user success</Translation>);
```
Comment (barnettZQG) [inline src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx:57]: User updated successfully.

File: src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+          placement="right"
+          width={800}
+          extButtons={this.showClickButtons()}
+          onClose={this.close}
+        >
+          <Form {...formItemLayout} field={this.field}>
+            <Row>
+              <Col span={12} style={{ padding: '0 8px' }}>
+                <FormItem label={<Translation>Name</Translation>} labelTextAlign="left" required>
+                  <Input
```
Comment (barnettZQG) [inline src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx:109]: There is no need for this field.

File: src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+                          pattern: checkName,
+                          message: <Translation>Please enter a valid name</Translation>,
+                        },
+                      ],
+                    })}
+                  />
+                </FormItem>
+              </Col>
+              <Col span={12} style={{ padding: '0 8px' }}>
+                <FormItem label={<Translation>Alias</Translation>}>
```
Comment (barnettZQG) [inline src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx:127]: There is no need for this field.

File: src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+                          },
+                        },
+                      ],
+                    })}
+                  />
+                </FormItem>
+              </Col>
+
+              <Col span={12} style={{ padding: '0 8px' }}>
+                <FormItem label={<Translation>ConfirmPassword</Translation>} required>
```
Comment (barnettZQG) [inline src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx:182]: For consistency, we do not set this field, but provide a button with an eye icon for showing the password.

File: src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+        .finally(() => {
+          this.setState({
+            isLoading: false,
+          });
+        });
+    });
+  };
+
+  showTitle() {
+    return i18n.t('Edit User');
```
Comment (barnettZQG) [inline src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx:70]: Reset the password and email for the administrator account.

File: src/pages/ApplicationList/index.tsx [hunk 6]
```diff
@@ -182,6 +231,9 @@ class Application extends Component<Props, State> {
             onClose={this.closeEditAppDialog}
           />
         </If>
+        <If condition={isEditAdminUser}>
+          <EditPlatFormUserDialog userInfo={userInfo} onClose={this.onCloseEditAdminUser} />
```
Comment (barnettZQG) [inline src/pages/ApplicationList/index.tsx:235]: Mv it to the TopBar component.

File: src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx (hunk 1 above)
Comment (wangbow) [reply src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx]: done

File: src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx (hunk 2 above)
Comment (wangbow) [reply src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx]: done.remove it

File: src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx (hunk 3 above)
Comment (wangbow) [reply src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx]: done.remove it

File: src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx (hunk 4 above)
Comment (wangbow) [reply src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx]: done.remove it ,and implement eye icon

File: src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx (hunk 5 above)
Comment (wangbow) [reply src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx]: done

File: src/pages/ApplicationList/index.tsx (hunk 6 above)
Comment (wangbow) [reply src/pages/ApplicationList/index.tsx:235]: done

Comment (barnettZQG) [reply src/pages/ApplicationList/index.tsx:235]: There is no change?

File: src/pages/ApplicationList/index.tsx [hunk 7]
```diff
@@ -110,10 +124,45 @@ class Application extends Component<Props, State> {
...
+    const { userInfo } = this.state;
+    const isAdminUser = this.isAdminUserCheck();
+    if (isAdminUser && userInfo && !userInfo.email) {
+      this.setState({
+        isEditAdminUser: true,
+      });
+    }
+  };
+
+  isAdminUserCheck = () => {
```
Comment (barnettZQG) [inline src/pages/ApplicationList/index.tsx:137]: The util function mv to the util package.

Comment (wangbow) [reply src/pages/ApplicationList/index.tsx]: done

File: src/pages/ApplicationList/index.tsx (hunk 6 above)
Comment (wangbow) [reply src/pages/ApplicationList/index.tsx]: done

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**